internal/cmd/workerpools/pool.go

```diff
@@ -0,0 +1,103 @@
+package workerpools
+
+import (
+	"fmt"
+	"github.com/spacelift-io/spacectl/internal/cmd"
+	"github.com/spacelift-io/spacectl/internal/cmd/authenticated"
+	"github.com/urfave/cli/v2"
+)
+
+type Pool struct {
+	ID          string  `graphql:"id" json:"id"`
+	Name        string  `graphql:"name" json:"name"`
+	Description *string `graphql:"description" json:"description"`
+	PendingRuns int     `graphql:"pendingRuns" json:"pendingRuns"`
+	BusyWorkers int     `graphql:"busyWorkers" json:"busyWorkers"`
+	Workers     []struct {
+		Id string `graphql:"id" json:"id"`
+	} `graphql:"workers" json:"workers"`
+}
+
+type poolJsonOutput struct {
+	ID                string  `json:"id"`
+	Name              string  `json:"name"`
+	Description       *string `json:"description"`
+	Space             string  `json:"space"`
```

Comment:
@jpJuni0r looks like there's still one place to remove. Everything else looks good though, so I'll merge as soon as this is removed.

Reply:
Goot catch, it's now removed.